src/main/scala-2.12/org/apache/james/gatling/jmap/rfc8621/JmapMailbox.scala

```diff
@@ -64,4 +66,24 @@ object JmapMailbox {
           .pause(1 second, 2 seconds)
       }
       .pause(5 second)
+
+  def saveStateAs(key: String): HttpCheck = jsonPath(statePath).saveAs(key)
+
+  def saveNewStateAs(key: String): HttpCheck = jsonPath(newStatePath).saveAs(key)
+
+  def getNewState(accountId: String = "accountId", oldState: String = "oldState"): HttpRequestBuilder = {
+    JmapHttp.apiCall("mailboxChanges")
+      .body(StringBody(
+        s"""{
+           |  "using": ["urn:ietf:params:jmap:core", "urn:ietf:params:jmap:mail"],
+           |  "methodCalls": [
+           |    ["Mailbox/changes", {
+           |      "accountId": "$${$accountId}",
+           |      "sinceState": "$${$oldState}"
+           |    }, "c1"]
```

Comment:
Can we get a back reference here? One to get created mailbox, one to get updated mailboxes?